addons/l10n_cn/views/report_voucher.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="external_layout_boxed" inherit_id="web.external_layout_boxed" primary="True">
```

Comment:
Is there a reason why we put a boxed layout? As normally, a report should work with any.

Reply:
@jco-odoo I did this according to the example provided by our Chinese partner. And according to my experience, I never see other layout used in China.